.clang-format

```diff
@@ -88,7 +88,7 @@ PenaltyBreakFirstLessLess: 120
 PenaltyBreakString: 1000
 PenaltyExcessCharacter: 1000000
 PenaltyReturnTypeOnItsOwnLine: 60
-PointerAlignment: Right
+PointerAlignment: Left
```

Comment:
What is the rationale behind the clang-format changes?

Reply:
We can also revert those changes, if we want to make this another PR. The rationale is easily explained. To align the pointer symbol on right will clearly show that a type if of pointer or reference type as in:
```c++
int a = 10;
int* b = &a;
```  
Now `b` is of type _pointer to integer_ thus the symbol that makes it a pointer is part of the type not part of the variable name - as it would be with right alignment. The same applies to references. 

The `QualifierAlignment` in line `123` makes sure we are using _east const_ which simply follows the binding of a qualifier to it's left as in:
```c++
int const * const c = &a
```
Where the _const_ qualifier always binds to it's left. If one would want to use _west const_ one has to write:
```c++
const int * const c = &a
```
Basically mixing both alignments (west & east) since it is not possible to write:
```c++
const int const *  c = &a
```